packages/docusaurus/src/client/exports/Image.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
```

Comment:
I wonder if we should rename this component. `Image` is actually a [HTML API](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image) e.g. `const img = new Image()`. So if someone imports this component without renaming, it's gonna override the the HTML API and potentially cause confusion.

But internally FB also calls its custom image React component `Image`, so I'm quite conflicted.

Unless we have a better name I think this is fine.